src/components/LoadingSpinner/LoadingSpinner.scss

```diff
@@ -1,3 +1,13 @@
+@use "sass:map";
+$contexts: (primary, secondary, critical, whiteSpinner);
```

Comment:
I think better to call the last one `invert` as this is the word used for icons displayed on an inverted BG. E.g. in a dark theme it will more like to be black then white.